ch_pipeline/analysis/flagging.py

```diff
@@ -139,6 +142,48 @@ def process(self, data) -> Union[containers.RFIMask, dcontainers.RFIMask]:
         return out
 
 
+class RFIStaticMask(task.SingleTask):
```

Comment:
This might be better with a more active name. `MaskStaticRFI` might be better. We haven't been very consistent before, but I think we maybe should get better on that front.